foreman/client.py

```diff
@@ -222,7 +247,8 @@ def __init__(self, foreman):
         """
         self._f = foreman
         # Preserve backward compatibility with old interface
-        for method_name in ('index', 'show', 'update', 'destroy', 'create'):
+        for method_name in ('index', 'show', 'update', 'destroy', 'create',
+                            'search'):
```

Comment:
it seems that it depends on foreman version, 
I tried with foreman 1.5.2:
>>> from foreman import client
>>> conn = client.Foreman(url="****", api_version=2, auth=("****", "****"))
>>> conn.operatingsystems.index()
{u'sort': {u'by': None, u'order': None}, u'search': None, u'results': [***********]}

and got results